transport/src/main/java/io/netty5/channel/DefaultChannelHandlerContext.java

```diff
@@ -1468,7 +1468,7 @@ public V call() throws Exception {
                     } finally {
                         if (e != null) {
                             Resource.dispose(value);
-                            logger.error(e);
+                            logger.error("Failed to dispose {}.", value, e);
```

Comment:
imho this could lead to undefined behaviour. We should call `value.toString()`, log it and then call `dispose(value)`.  This is needed as the logger might be async and also because dispose might cause trouble when calling toString later.